### ISSUE TITLE
Collision problem

### ISSUE THREAD

**Issue by Crinouche** [Mogli12/AIVehicleExtension#147]
Hello Mogli,

I have a problem with AIVE V.2.4 or earlier.
The combine harvester detects a collision with the tractor when it approaches to empty. It stops (while I have selected "do not stop to unload") and does not restart when it is empty. I also unhitched the collisions, in the hud
I have to stop AIVE, move a little forward and reactivate AIVE. I removed all mods and the problem remains the same.
Thank you and good job !!https://gist.github.com/Crinouche/c868dbefb25d13c4fdbf168f2020bc1f/revisions#diff-ffa9d213659f6ecce35617dee1fc8517[](url)

**Comment by Mogli12** [comment]
Hi @Crinouche 

I was not able to reproduce it with all vehicles. Which combine harvester did you use? I suspect that this might be caused by a built in collision box and not by the collision box of AIVE.

Regards,
Stefan

**Comment by Crinouche** [comment]
Hi,
I use the JD S790.
But I just tried with the NH CR 1090, the MF 7347 and the AGCO Ideal and the problem is the same, as soon as the tractor approaches, the combine harvester stops, empty and never leaves.
This tells me on the upper right that there is an obstacle and that the worker is blocked.

**Comment by 51stHD** [comment]
I have encountered the same problem with the small Rostselmash Nova 330 harvester.

Additionally, when using two of these harvesters and the 2nd harvester catches up with the 1st one, everything locks up, the tractor I'm in drives on it's own across the map, the money figure increases on it's own and I have to crash out of the game.

This also happens regardless of the collision setting in AIVE. Sorry, I did't keep a copy of the log.
